addon/enums/currency-symbol-type.js

```diff
@@ -1,8 +0,0 @@
-import Enumerable from './enumerable';
```

Comment:
My two cents would be not erasing this one: this is used in practically **all** of our projects, and I don't think we have the time to migrate them just now...

Reply:
Well if we have the time to upgrade them to this version we should have the time to upgrade this... it's minor compared to the upgrade of ember cli uniq...

Reply:
Yes... But I don't know if we want to do that right now. And this will block all future versions of this add-on. I would prefer keeping them both, talk about it in the sync, and start migrating when we have time.

Reply:
But, in the end, it's not my decision cc @tiagobrito